### PR TITLE
WEBDEV-5316 - Mediatype-icon color config update

### PR DESCRIPTION
**Pull request by dualcnhq**
## Changes:

- update fillColor
- add quick fix for itemTile title `title` attribute display

### CODE REVIEW

File: src/tiles/mediatype-icon.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class MediatypeIcon extends LitElement {
       }
 
       .fill-color {
-        fill: var(--iconCustomFillColor, var(--iconFillColor, '#000000'));
+        fill: var(--iconFillColor, '#000000');
```
Comment (mc2) [inline src/tiles/mediatype-icon.ts:71]: Blocker: Now that the collection icon has a fill color, we can get rid of `private get collectionColor()` and `--iconCustomFillColor` in tile-list.ts and tile-list-compact.ts

Need to handle this in the same branch.

**Review by mc2 — CHANGES_REQUESTED** [review]

File: src/tiles/mediatype-icon.ts (hunk 1 above)
Comment (dualcnhq) [reply src/tiles/mediatype-icon.ts:71]: updated now...

File: src/tiles/list/tile-list-compact.ts [hunk 2]
```diff
@@ -45,7 +45,6 @@ export class TileListCompact extends LitElement {
           <mediatype-icon
             .mediatype=${this.model?.mediatype}
             .collections=${this.model?.collections}
-            style="--iconCustomFillColor: ${ifDefined(this.collectionColor)}"
```
Comment (mc2) [inline src/tiles/list/tile-list-compact.ts:48]: Great, thanks Clau!

File: src/assets/img/icons/mediatype/account.ts [hunk 3]
```diff
@@ -1,14 +1,13 @@
 import { svg } from 'lit';
 
 export const accountIcon = svg`
-  <svg viewBox="0 0 100 100"
-    xmlns="http://www.w3.org/2000/svg">
+  <svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
+    <title>icon/account</title>
```
Comment (mc2) [inline src/assets/img/icons/mediatype/account.ts:5]: blocker: We don't know what the `<title>`s should be. And there are conflicting styles.

Let's not make any changes in these, but back out the changes in all the mediatype icons and wait for the right design.

There needs to be a design ticket for this. I will write it.

Comment (mc2) [reply src/assets/img/icons/mediatype/account.ts:5]: Let's try to center each pull request on the (hopefully) one issue of each Jira ticket. 
Here: "Item tile icon stats color"

If we do get Jira tickets are not clear and focused, let's all include in our development process: sending them back, i.e. commenting that the writer of the ticket needs to 
- be clearer
- break up the ticket
- etc.

**Comment by mc2** [comment]
@dualcnhq Please note the Jira number in the first comment of every PR. In GitLab, we do it in the "description"

File: src/styles/item-tile-styles.ts [hunk 4]
```diff
@@ -12,16 +12,13 @@ export const baseItemTileStyles = css`
     flex-direction: column;
```
Comment (mc2) [inline src/styles/item-tile-styles.ts:12]: blocker: I'm not sure if we have a coding standard for whitespace in .css definitions. But we do want whitespace between each selector and rule set. Please undo this change.

Again, `yarn format` should do this, but I'm not sure. Please `yarn format` before every commit in this project.

**Review by mc2 — CHANGES_REQUESTED** [review]

File: src/styles/item-tile-styles.ts (hunk 4 above)
Comment (mc2) [reply src/styles/item-tile-styles.ts:12]: Not resolved.

Please move file to `src/tiles/grid/` next to `item-tile.ts`

**Review by mc2 — CHANGES_REQUESTED** [review]

**Review by mc2 — APPROVED** [review]
